src/components/Header/Header.tsx

```diff
@@ -32,10 +33,10 @@ const Header = ({ nav }: Props) => {
         </Button>
 
         <img src="/assets/logo.png" alt="logo" width="80" className="logo" />
-        <Link href="" variant="text" color="black">
+        <NextLink href="/login" color="black">
```

Comment:
This looks a bit weird. The `color` property doesn't exist on `NextLink`, right?

I think you need to remove it.